### PR TITLE
Feat/liquidations

### PR DESCRIPTION
**Pull request by SirTLB**
## Description and Motivation
This PR introduces Liquidation loops to the bot. Meaning it is now possible to track and trace borrow/lending platforms that are derived from Anchor Forks and, if a position is above max LTV, liquidate that person and receive a small reward for doing so. It is not yet possible to receive the collateral assets, as they work with a bidding auction system. 

## Related Issues
#94 

---
## Checklist:

- [x] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-bots/blob/main/CONTRIBUTING.md).
- [x] My pull request has a sound title and description (not something vague like `Update index.md`)
- [x] All existing and new tests are passing (if any).
- [x] I updated/added relevant documentation.
- [x] The code is formatted properly using eslint

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,140 +1,42 @@
...
-import { SkipLoop } from "./core/types/arbitrageloops/skipLoop";
-import { setBotConfig } from "./core/types/base/botConfig";
-import { LogType } from "./core/types/base/logging";
-import { removedUnusedPools } from "./core/types/base/pool";
+import { DexLoop } from "./core/types/arbitrageloops/loops/dexloop";
+import { LiquidationLoop } from "./core/types/arbitrageloops/loops/liqMempoolLoop";
+import { DexConfig, LiquidationConfig, setBotConfig, SetupType } from "./core/types/base/configs";
 // load env files
-dotenv.config();
+dotenv.config({ path: "./src/envs/terra.env" });
```
Comment (nick134-bit) [inline src/index.ts:9]: hardcoded .env?

File: src/index.ts [hunk 2]
```diff
@@ -1,140 +1,42 @@
...
-			paths,
-			trySomeArb,
-			getPoolStates,
-			getFlashArbMessages,
-			chainOperator,
-			botConfig,
-			logger,
-			[...paths],
-		);
+	if (botConfig.setupType === SetupType.DEX) {
```
Comment (nick134-bit) [inline src/index.ts:27]: case usable?

`switch (botConfig.setupType) {`

		case SetupType.DEX:
			loop = await DexLoop.createLoop(chainOperator, <DexConfig>botConfig, logger);
			//print the created arbitrage loop
			await logger.defaults.logDexLoop(loop)
			break;
		case SetupType.LIQUIDATION:
			loop = await LiquidationLoop.createLoop(chainOperator, <LiquidationConfig>botConfig, logger);
			//print the created liquidation loop
			await logger.defaults.logLiqLoop(loop);
			break;
	}
`

**Review by nick134-bit — APPROVED** [review]
very nice work! its so much cleaner now

File: src/index.ts (hunk 2 above)
Comment (SirTLB) [reply src/index.ts]: done, nice addition

**Pull request merged**